lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -130,6 +130,13 @@ def expected_path(constant)
         end
 
         def camel_to_snake_case(string)
+          if defined?(ActiveSupport::Inflector)
+            if File.exist?('./config/initializers/inflections.rb')
+              require './config/initializers/inflections'
+            end
```

Comment:
I’m afraid this code is a bit slow. Could we do the `defined?`/`File.exists?`/`require` thing just _once_ and store the result perhaps a class variable?